libvast/vast/detail/tracepoint.hpp

```diff
@@ -239,7 +239,7 @@
 
 
 #if defined(__ELF__) && (defined(__x86_64__) || defined(__i386__)) && \
-    !VAST_DISABLE_SDT
+    VAST_ENABLE_SDT
```

Comment:
Please don't forget to provide this macro by default if you flip the semantics here.

Reply:
Already happening via `target_compile_definitions`